orc8r/cloud/helm/orc8r/Chart.yaml

```diff
@@ -13,7 +13,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for magma orchestrator
 name: orc8r
-version: 1.5.0
+version: 1.5.3
```

Comment:
1.5.1,1.5.2 are taken up by a couple of PRs of mine that haven't been merged yet.